CMakeLists.txt

```diff
@@ -72,6 +72,8 @@ endif ()
   include(AddMLIR)
   include(HandleLLVMOptions)
 
+  set(MLIR_BINARY_DIR ${PROJECT_SOURCE_DIR}/llvm/build)
```

Comment:
Isn't `llvm/build` dependent on how one configured the LLVM build?

Reply:
`LLVMConfig.cmake` makes a suitable variable available via `find_package`, whereas `MLIRConfig.cmake` does not. I pushed a workaround, but this should probably be fixed upstream, right?

Reply:
Yes, please :)